docs/Ruleset.md

```diff
@@ -29,10 +29,10 @@ V010 | MAJOR | Class public method parameter added
 V011 | MAJOR | Class protected method parameter added
 V012 | MAJOR | *New public constructor (does not match supertype)*
 V013 | MAJOR | *New protected constructor (does not match supertype)*
-V015 | MAJOR | Class public method added
```

Comment:
The reason this is a major breaking change is because someone else could be extending a class and has defined a different signature of the method that's now being added. This is why the equivalent for a final class is not major.

Reply:
Same reasoning for traits, but instead of extending, using the trait in a class that would already have said methods.